src/Form/SendAnEmailType.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class SendAnEmailType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('emailRecipient', EmailType::class, ["label" => "TO: "])
+            ->add('subject', TextType::class, ["label" => "Subject: "])
+            ->add('emailSender', EmailType::class, ["label" => "FROM: "])
+            ->add('message', TextType::class, ["label" => "Message: "])
+            ->add('submit', SubmitType::class, ["label" => "Submit"])
+        ;
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            ["novalidate" => "novalidate"]
```

Comment:
Write your own validation first to test it is working then immediately remove this option to ensure that the default HTML-5 validation is in place with my validation.